### PR TITLE
Change the instructions in favor of local storage for Prometheus and AM

### PR DESCRIPTION
**Pull request by rh-max**

### CODE REVIEW

**Comment by rh-max** [comment]
@ahardin-rh Could you please review this? (also to be merged to enterprise-4.2)

File: modules/monitoring-configuring-a-local-persistent-volume-claim.adoc [hunk 1]
```diff
@@ -59,15 +57,17 @@ data:
...
+          storageClassName: *local-storage*
           resources:
             requests:
               storage: *40Gi*
 ----
 +
-And to configure a PVC that claims any configured {product-title} block PV as a persistent storage for Alertmanager, you can use:
+In the above example, the storage class created by the Local Storage Operator is called `local-storage`.
++
+And to configure a PVC that claims local persistent storage for Alertmanager, use:
```
Comment (ahardin-rh) [inline modules/monitoring-configuring-a-local-persistent-volume-claim.adoc]: s/And to/To

File: monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc [hunk 2]
```diff
@@ -34,11 +34,13 @@ In {product-title} 4.1 deployed on bare metal, Prometheus and Alertmanager canno
...
-* Unless you enable dynamically-provisioned storage, make sure you have a Persistent Volume (PV) ready to be claimed by the Persistent Volume Claim (PVC), one PV for each replica. Since Prometheus has two replicas and Alertmanager has three replicas, you need five PVs to support the entire monitoring stack.
+* Dedicate sufficient local persistent storage to ensure that the disk does not become full. How much storage you need depends on the number of pods. For information on system requirements for persistent storage, see xref:../../scalability_and_performance/scaling-cluster-monitoring-operator.adoc#prometheus-database-storage-requirements[Prometheus database storage requirements].
+* Make sure you have a Persistent Volume (PV) ready to be claimed by the Persistent Volume Claim (PVC), one PV for each replica. Since Prometheus has two replicas and Alertmanager has three replicas, you need five PVs to support the entire monitoring stack. The Persistent Volumes should be available from the Local Storage Operator. This does not apply if you enable dynamically-provisioned storage.
```
Comment (ahardin-rh) [inline monitoring/cluster-monitoring/configuring-the-monitoring-stack.adoc]: s/dynamically-provisioned/dynamically provisioned

**Comment by ahardin-rh** [comment]
Just a few minor comments. Otherwise, looks great!

**Comment by rh-max** [comment]
@ahardin-rh Thank you! Last commit implements those two suggestions.

**Comment by ahardin-rh** [comment]
@rh-max Can you please squash? Then, I will merge and cherry-pick :cherries:

**Comment by rh-max** [comment]
@ahardin-rh Sure, done! I keep forgetting to squash...

File: modules/monitoring-configuring-a-local-persistent-volume-claim.adoc [hunk 3]
```diff
@@ -40,12 +38,12 @@ data:
           storageClassName: *_storage class_*
           resources:
             requests:
-              storage: *_40Gi_*
+              storage: *_amount of storage_*
```
Comment (juzhao) [inline modules/monitoring-configuring-a-local-persistent-volume-claim.adoc]: metadata:
      name: *_PVC name_*
actually it is not PVC name, it is PVC name prefix
eg:
 if name: prometheus for prometheusK8s
the pvc would be
prometheus-prometheus-k8s-0
prometheus-prometheus-k8s-1

Comment (rh-max) [reply modules/monitoring-configuring-a-local-persistent-volume-claim.adoc]: Implemented in #16978.

Comment (jboxman) [inline modules/monitoring-configuring-a-local-persistent-volume-claim.adoc]: We aren't using this styling for our YAML annotations (https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

Comment (rh-max) [reply modules/monitoring-configuring-a-local-persistent-volume-claim.adoc]: Thank you, implemented this in #16978.

**Pull request merged**